examples/gerry_learning_auditing.py

```diff
@@ -0,0 +1,46 @@
+# testing integration of gerry fair with aif360
+import sys
+sys.path.append("../")
+from aif360.algorithms.preprocessing.optim_preproc_helpers.data_preproc_functions import load_preproc_data_adult
```

Comment:
maybe you can try with the german dataset (it's much smaller so it should run quickly)

Reply:
added in subsampling of the dataset to address this point.